entrypoint.sh

```diff
@@ -4,7 +4,8 @@ echo -e "========================================="
 echo -e "=   WELCOME TO THE RIGETTI FOREST SDK   ="
 echo -e "========================================="
 echo -e "Copyright (c) 2016-2019 Rigetti Computing\n"
-/src/quilc/quilc --quiet -R &
-/src/qvm/qvm --quiet -S &
+
+/src/quilc/quilc --quiet --check-sdk-version=no -R 2> quilc.log &
+/src/qvm/qvm --quiet --check-sdk-version=no -S 2> qvm.log &
```

Comment:
I think all logging should go to stderr, but is there any reason not to redirect stdout as well given we're running in server mode?

Reply:
no reason, happy to put both in the log